ios/Surveys/Resources/Assets/Assets.xcassets/logoWhite.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "filename" : "Logo White.pdf",
+      "idiom" : "universal"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
+}
```

Comment:
should we enable preserve vector data for this PDF? if so we won't need care about scaling images when their size changed 😄 .
Same for all PDFs below.

Reply:
Good catch!

Reply:
Fixed here: https://github.com/suho/kmm-ic/pull/78/commits/37bd5a9a2e6a12f1d1fa04ec84d001038ab23f88